aten/src/ATen/native/native_functions.yaml

```diff
@@ -274,6 +274,18 @@
   use_c10_dispatcher: full
   variants: function
 
+- func: sgn(Tensor self) -> Tensor
+  use_c10_dispatcher: full
+  variants: function, method
+
+- func: sgn_(Tensor(a!) self) -> Tensor(a!)
+  variants: method
+
+- func: sgn.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
+  dispatch:
+    CPU: sgn_out
```

Comment:
Is this dispatch required or will the system automatically match these functions?

Reply:
Dispatch is not required, `sgn_out` function is the same for CPU and CUDA backends, actual backend selection is handled by stubs.

Reply:
you are right, it's not needed. removed